assets/src/edit-story/components/checklist/constants.js

```diff
@@ -469,6 +475,33 @@ export const PRIORITY_COPY = {
       `${MAX_VIDEO_RESOLUTION}p`
     ),
   },
+  ampValidation: {
+    title: __('Compatibility', 'web-stories'),
+    footer: (
+      <TranslateWithMarkup
+        mapping={{
+          a: (
+            <Link
+              href="https://wordpress.org/plugins/web-stories/#what%20do%20i%20do%20about%20an%20amp%20validation%20issue%20in%20search%20console%3F"
+              rel="noreferrer"
+              target="_blank"
+              aria-label={__(
+                'Web Stories frequently asked questions',
+                'web-stories'
+              )}
+              onClick={onAmpErrorClick}
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+            />
+          ),
+        }}
+      >
+        {__(
+          'Your story contains errors that can affect eligibility to appear on Google. This can be caused by incompatible WordPress plugins, web application firewalls, or other hosting configuration issues. Please see <a>our FAQ</a> for more information and debugging tips.',
```

Comment:
This sentence is a) quite long and b) contains more info than the FAQ entry 😅 

Shouldn't we just link to the forums instead in that case?

Reply:
😂 

Shortened to this? 
> Your story contains errors that can affect eligibility to appear on Google. Please see our FAQ for more information.